ResolveCurrentProjectStaticWebAssetsInputsDependsOn -> MSB4057: The target "TypeScriptCompile" does not exist in the project.

According to https://docs.microsoft.com/en-us/aspnet/core/razor-pages/ui-class?view=aspnetcore-3.0&tabs=visual-studio#create-an-rcl-with-static-assets I added `<ResolveCurrentProjectStaticWebAssetsInputsDependsOn>TypeScriptCompile;$(ResolveCurrentProjectStaticWebAssetsInputs)</ResolveCurrentProjectStaticWebAssetsInputsDependsOn>` but only get:

> C:\Program Files\dotnet\sdk\3.0.100\Sdks\Microsoft.NET.Sdk.Razor\build\netstandard2.0\Microsoft.NET.Sdk.Razor.StaticWebAssets.targets(266,61): error MSB4057: The target "TypeScriptCompile" does not exist in the project.

This single xy.ts file perfectly compiles to wwwroot\xy.js but only gets included after second build -- which CI of course does not.

What is missing? Entire csproj file below:

```xml
<Project Sdk="Microsoft.NET.Sdk.Razor">

  <PropertyGroup>
    <TargetFramework>netstandard2.0</TargetFramework>
    <RazorLangVersion>3.0</RazorLangVersion>
  </PropertyGroup>

  <PropertyGroup>
    <TypeScriptTarget>es2017</TypeScriptTarget>
    <!--<TypeScriptStrict>true</TypeScriptStrict>-->
    <TypeScriptAlwaysStrict>true</TypeScriptAlwaysStrict>
    <TypeScriptPreserveConstEnums>true</TypeScriptPreserveConstEnums>
    <TypeScriptNoEmitOnError>true</TypeScriptNoEmitOnError>
    <TypeScriptRemoveComments>true</TypeScriptRemoveComments>
    <TypeScriptOutDir>wwwroot</TypeScriptOutDir>

    <!-- see https://docs.microsoft.com/en-us/aspnet/core/razor-pages/ui-class?view=aspnetcore-3.0&tabs=visual-studio#create-an-rcl-with-static-assets -->
    <ResolveCurrentProjectStaticWebAssetsInputsDependsOn>TypeScriptCompile;$(ResolveCurrentProjectStaticWebAssetsInputs)</ResolveCurrentProjectStaticWebAssetsInputsDependsOn>
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="Microsoft.AspNetCore.Components.Web" Version="3.0.0" />
    <PackageReference Include="Microsoft.TypeScript.MSBuild" Version="3.6.3">
      <PrivateAssets>all</PrivateAssets>
      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
    </PackageReference>
  </ItemGroup>

</Project>
```


---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 9fd019c7-ceb7-6ef3-26ba-6b44341b6fc5
* Version Independent ID: fd116d00-0dc7-efaf-c714-34d77b345a05
* Content: [Reusable Razor UI in class libraries with ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/razor-pages/ui-class?view=aspnetcore-3.0&tabs=visual-studio#create-an-rcl-with-static-assets)
* Content Source: [aspnetcore/razor-pages/ui-class.md](https://github.com/aspnet/AspNetCore.Docs/blob/master/aspnetcore/razor-pages/ui-class.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-razorpages**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

--- a/aspnetcore/razor-pages/ui-class.md
+++ b/aspnetcore/razor-pages/ui-class.md
@@ -115,10 +115,10 @@ To include TypeScript files in an RCL:
 1. Include the TypeScript target as a dependency of the `ResolveCurrentProjectStaticWebAssets` target by adding the following target inside of a `PropertyGroup` in the project file:
 
    ```xml
-   <ResolveCurrentProjectStaticWebAssetsInputsDependsOn>
-     TypeScriptCompile;
-     $(ResolveCurrentProjectStaticWebAssetsInputs)
-   </ResolveCurrentProjectStaticWebAssetsInputsDependsOn>
+  <ResolveCurrentProjectStaticWebAssetsInputsDependsOn>
+    CompileTypeScript;
+    $(ResolveCurrentProjectStaticWebAssetsInputs)
+  </ResolveCurrentProjectStaticWebAssetsInputsDependsOn>
    ```
 
 ### Consume content from a referenced RCL

